### PR TITLE
Improve promise code

### PR DESCRIPTION
**Pull request by superandrew213**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ var IS_WINDOWS = '<\/Receipt>';
 
 var amazon;
 
+function handlePromisedFunctionCb(error, response) {
+	if(error) {
```
Comment (voltrue2) [inline index.js]: Would you please add a space between `if` and its `(`?

**Comment by voltrue2** [comment]
Hello
Sorry for the really late response.
I have a very small comment on the PR, if you don't mind handling it.
Once that is resolved, I believe we are ready to merge!

Cheers

**Comment by superandrew213** [comment]
No problem at all :). Done!

**Comment by voltrue2** [comment]
Merged!
Cheers.

There is another PR that I would like to merge before publishing this version of the code to npm.
Hopefully we get to push this out soon!

**Comment by superandrew213** [comment]
Sure, no problem!

**Pull request merged**